src/test/resources/processor/UploadAsymmetricModelRequestBody.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "traced_small_model",
```

Comment:
I think we also need introduce remote model integ tests for this feature.

Maybe you can add in this or separate PR.

Reply:
We have `./gradlew remoteModelIntegTest` for remote model only integ test, so that `./gradlew integTest` won't trigger it.

You can follow: https://github.com/opensearch-project/neural-search/blob/main/DEVELOPER_GUIDE.md#running-tests-with-remote-models

And finally your test should be picked in the remote model integ test CI as well, like: https://github.com/opensearch-project/neural-search/actions/runs/19308732819/job/55223053982?pr=1605